README.md

```diff
@@ -16,3 +16,86 @@ The project and Python dependencies can be installed from source with:
 ```
 pip3 install .
 ```
+
+## Configuration System
+
+This project provides a configuration system that supplies options (e.g., user name, password) to a command-line interface ([CLI](https://en.wikipedia.org/wiki/Command-line_interface)) tool.
+The configuration system follows a tiered order, allowing options to be specified and overridden from both files and command-line options.
+
+### Configuration Sources
+
+In addition to CLI options, the configuration system loads options from [JSON](https://en.wikipedia.org/wiki/JSON) files located across multiple directories.
+By default, configuration files are named `edq-config.json`.
+This value is customizable, but this document will assume the default is used.
+
+For example, a configuration file containing the `user` and `pass` options might look like this:
+```
+{
+    "user": "edq-user",
+    "pass": "1234567890"
+}
+```
+
+The table below lists the configuration sources in the order they are evaluated.
+All sources are processed in the order shown in the table. 
+The value from the later source in the table overrides the earlier one if there are multiple sources.
+
+| Source   | Description |
+| :-----:  | :---------- |
+| Global   | The global configuration file path is a platform-specific user location by default. |
```

Comment:
You don't need the "by default" part.
We will discuss how it can be overwritten later, so we don't need to add that complication here.